android-kit-base/src/test/kotlin/com/mparticle/kits/KitConfigurationTest.kt

```diff
@@ -676,6 +678,97 @@ class KitConfigurationTest {
         )
     }
 
+    @Test
+    fun testConvertToSparseArray() {
+        val kitConfiguration = MockKitConfiguration()
+        val jsonData = """   
+        {
+           "7456529": 0,
+          "10887805": 0,
+          "13992010": 0,
+          "15360852": 0,
+          "17455322": 0,
+          "18683141": 0,
+          "23029959": 0,
+          "41851400": 0,
+          "47355425": 0,
+          "54925556": 0,
+          "56409892": 0,
+          "66701264": 0
+         }
+        """.trimIndent()
+        val jsonConfiguration = JSONObject(jsonData)
+        val method: Method = MockKitConfiguration::class.java.getDeclaredMethod("convertToSparseArray", JSONObject::class.java)
+        method.isAccessible = true
+        val result = method.invoke(kitConfiguration, jsonConfiguration) as SparseBooleanArray
+        Assert.assertEquals(12, result.size())
+    }
+
+    @Test
+    fun testConvertToSparseArray_When_JSON_OBJECT_IS_NULL() {
```

Comment:
Isn't this JSON object just an empty object as opposed to a `null`?  Should there be a test case for 
`{"ec" : null}` or no?

Reply:
Add unit test case for null validation